nighthawk/source/client/options.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "google/protobuf/duration.proto";
+
+message CommandLineOptions {
```

Comment:
Ultimately this will be a load test specification, the fact that it can arrive via CLI is just a concrete detail. Consider changing to reflect this or leave a TODO and we can do it later.